ReadME.md

```diff
@@ -34,19 +34,19 @@ Rezume is a free online portal for generating resumes without worrying about the
-  1. [Prettier - Code formatter](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) helps you to format your JavaScript / TypeScript / CSS using [Prettier](https://github.com/prettier/prettier).
-  2. [Cobalt2 Theme Official](https://marketplace.visualstudio.com/items?itemName=wesbos.theme-cobalt2) is the theme used by Wes Bos, which comes with a nice user experience and a lot of awesome features. I am using a modified version of Cobalt2 that's a crossover between [Monokai](https://www.monokai.pro/) and Cobalt2.
-  3. [GitLens - Git supercharged](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens) supercharges the Git capabilities built into Visual Studio Code. It helps you to visualize code authorship at a glance via Git blame annotations and code lens, seamlessly navigate and explore Git repositories, gain valuable insights via powerful comparison commands, and so much more.
-  4. [Git History](https://marketplace.visualstudio.com/items?itemName=donjayamanne.githistory) helps you to view git log, file history, compare branches or commits inside VS Code.
-  5. [ESLint](https://marketplace.visualstudio.com/items?itemName=dbaeumer.vscode-eslint) integrates ESLint into VS Code. If you are new to ESLint check the [documentation](http://eslint.org/).
-  6. [ES7 React/Redux/GraphQL/React-Native snippets](https://marketplace.visualstudio.com/items?itemName=dsznajder.es7-react-js-snippets) provides you JavaScript and React/Redux snippets in ES7 with Babel plugin features for VS Code.
-  7. [Todo Tree](https://marketplace.visualstudio.com/items?itemName=Gruntfuggly.todo-tree) `(optional)` quickly searches (using ripgrep) your workspace for comment tags like TODO and FIXME, and displays them in a tree view in the explorer pane. Clicking a TODO within the tree will open the file and put the cursor on the line containing the TODO.
-  8. [`vscode-icons`](https://marketplace.visualstudio.com/items?itemName=vscode-icons-team.vscode-icons) `(optional)` is more of a cosmetic thing, which brings icons to the Visual Studio Code editor.
+- [ ] [Git](https://git-scm.com/downloads)
+- [ ] [Node.js](https://nodejs.org/en/) (Please go with any latest LTS versions)
+- [ ] [Linux Based Terminal](https://gitforwindows.org/), if you're using Windows and installed with Git.
+- [ ] [Google Chrome](https://www.google.com/chrome/) or a decent latest web browser.
+- [ ] [Visual Studio Code](https://code.visualstudio.com/)** is the go-to editor for some complex languages like React using JSX or TypeScript and I finally agreed with [Wes Bos](https://wesbos.com/) that VS Code is better for development of React JS. Obviously, with VS Code, you should be having the right extensions to support your React JS development, but comparing with Sublime Text 2, VS Code is still in its infancy. Get these extensions and here's my [dotfile](https://gist.github.com/praveenscience/ebb5439f31774ad2fdc14cb9e7de1fc0):
```

Comment:
Supposed to be bold. That's wrong. Can you fix?

![image](https://user-images.githubusercontent.com/1830380/161397993-a994f8cd-9793-4be2-a16c-3f4609721c6d.png)

Reply:
Yes, will do that